src/Promenade.Web/Views/Shared/_Layout.cshtml

```diff
@@ -6,6 +6,8 @@
     <title>@ViewData["Title"]</title>
     <link rel="stylesheet" href="~/css/main.min.css" asp-append-version="true" />
     <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
+    <link rel="shortcut icon" href="#" />
```

Comment:
Enter an issue for shortcut icons if we're disabling them here.

Reply:
There's a task in Basecamp about it.